### PR TITLE
fix: add NULL as an extra bin

### PR DESCRIPTION
**Pull request by almeidabbm**
Closes: [#9845](https://github.com/lightdash/lightdash/issues/9845)

### Description:

- Custom range:
image

- Fixed bins:
image

- Fixed width - was already working without any change:
image

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Review by rephus — APPROVED** [review]
LGTM 

We are not showing tooltips or axisLabels on null dimensions. But I guess this is not something we've introduced in this PR, so I'm happy to fix it on a separate PR . 

image

**Comment by almeidabbm** [comment]
> LGTM
> 
> We are not showing tooltips or axisLabels on null dimensions. But I guess this is not something we've introduced in this PR, so I'm happy to fix it on a separate PR .
> 
> image

I have made a fix for this as it seemed like a quick thing in this [commit](https://github.com/lightdash/lightdash/pull/9866/commits/dc518c63edcc29b6e7f86ee1f477b3c8d4a665bb)

Not sure if there could be any side effects tho 🤔 but don't think so as we're only running a fallback to formatted if raw doesn't exist - by using a `nullish coalesce`

File: packages/common/src/index.ts [hunk 1]
```diff
@@ -775,12 +775,12 @@ export type UpdateProject = Omit<
...
     rows: ResultRow[],
-    onlyRaw: boolean = false,
+    preferRaw: boolean = false,
 ): Record<string, unknown>[] =>
     rows.map((row) =>
         Object.keys(row).reduce<Record<string, unknown>>((acc, key) => {
-            const value = onlyRaw
-                ? row[key]?.value.raw
+            const value = preferRaw
+                ? row[key]?.value.raw ?? row[key]?.value.formatted
```
Comment (rephus) [inline packages/common/src/index.ts:783]: I thought linter was not happy about this nested conditions

Comment (almeidabbm) [reply packages/common/src/index.ts]: Same, I have changed this so either are assigned to a variable and easier to read 👌

**Comment by TuringLovesDeathMetal** [comment]
I feel like extra group looks right 👀 nice one!

**Pull request merged**